### PR TITLE
Llc outputs error_signals

### PR DESCRIPTION
**Pull request by SwayStar123**
## Checklist

- [ ] I have linked to any relevant issues.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation where relevant (API docs, the reference, and the Sway book).
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added (or requested a maintainer to add) the necessary `Breaking*` or `New Feature` labels where relevant.
- [ ] I have done my best to ensure that my PR adheres to [the Fuel Labs Code Review Standards](https://github.com/FuelLabs/rfcs/blob/master/text/code-standards/external-contributors.md).
- [ ] I have requested a review from the relevant team or maintainers.

### CODE REVIEW

File: sway-lib-std/src/error_signals.sw [hunk 1]
```diff
@@ -1,18 +1,18 @@
 //! Values which signify special types of errors when passed to `std::revert::revert`.
 library;
 
-/// Revert with this value for a failing call to `std::revert::require`.
-/// 18446744073709486080
+/// A revert with this value signals that it was caused by a failing call to `std::revert::require`.
+/// The value is: 18446744073709486080
```
Comment (bitzoic) [inline sway-lib-std/src/error_signals.sw:5]: I'd probably put this under additional information

File: sway-lib-std/src/outputs.sw [hunk 2]
```diff
@@ -90,18 +193,62 @@ pub fn output_amount(index: u64) -> u64 {
...
+/// * [Option<b256>] - The reciever of the output if it is a `Output::Coin`. None otherwise.
+///
+/// # Panics
+///
+/// * When the output type is unrecognized. This should never happen.
+///
+/// # Examples
+///
+/// ```sway
+/// use std::outputs::output_reciever;
```
Comment (bitzoic) [inline sway-lib-std/src/outputs.sw:245]: The function here is `output_asset_to()`

**Review by bitzoic — CHANGES_REQUESTED** [review]

File: sway-lib-std/src/error_signals.sw (hunk 1 above)
Comment (SwayStar123) [reply sway-lib-std/src/error_signals.sw:5]: The value?

Comment (bitzoic) [reply sway-lib-std/src/error_signals.sw:5]:
```suggestion
/// A revert with this value signals that it was caused by a failing call to `std::revert::require`.
/// 
/// # Additional Information
///
/// The value is: 18446744073709486080
```

**Review by bitzoic — APPROVED** [review]

**Pull request merged**